### PR TITLE
Add Packet Wavelength filter to LIV Plot

### PR DESCRIPTION
**Pull request by sarthak-dv**
### :pencil: Description

**Type:** :rocket: `feature` 

Added a Wavelength range filter to restrict the analysis of interacting packets in LIV Plot.

### :pushpin: Resources

Examples, notebooks, and links to useful references.

### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [ ] Other method (describe)
- [ ] My changes can't be tested (explain why)

### :ballot_box_with_check: Checklist

- [ ] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.

### CODE REVIEW

**Comment by sarthak-dv** [comment]
@jamesgillanders @jaladh-singhal  I have added the species not found in wvl range logger, please review this PR.

File: docs/io/visualization/how_to_liv_plot.ipynb [hunk 1]
```diff
@@ -117,6 +117,32 @@
     "plotter.generate_plot_mpl(packets_mode=\"real\")"
```
Comment (jaladh-singhal) [inline docs/io/visualization/how_to_liv_plot.ipynb:117]: <p>I think this got left unaddressed from your last PR: it should be "...regroup the bins with broader widths..." - don't we throw warning/error for num_bins &gt; num_shells?</p><p> </p><p>Same for description in mpl cell above</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2784/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Comment (jaladh-singhal) [inline docs/io/visualization/how_to_liv_plot.ipynb:117]: <p>is this intentional or a typo?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2784/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

**Review by jaladh-singhal — APPROVED** [review]
Looks good to me - some minor comments for notebook

File: docs/io/visualization/how_to_liv_plot.ipynb (hunk 1 above)
Comment (sarthak-dv) [reply docs/io/visualization/how_to_liv_plot.ipynb:117]: <p>It is a typo, I was testing the output and forgot to change it back, thanks for pointing it out.</p>

Comment (sarthak-dv) [reply docs/io/visualization/how_to_liv_plot.ipynb:117]: <p>Yeah it should be "...regroup the bins with broader widths...", I changed the code but forgot to change the docs in the notebook. </p>

**Comment by sarthak-dv** [comment]
> Looks good to me - some minor comments for notebook

Thanks for noticing the errors I overlooked, I have made the corrections please review it again.

**Review by jaladh-singhal — APPROVED** [review]
Thanks

**Review by jamesgillanders — APPROVED** [review]
Great work Sarthak! Code is concise and tidy -- happy to merge.

**Pull request merged**